src/App.js

```diff
@@ -10,17 +10,88 @@ class App extends Component {
 
     this.state = {
       smoothies: smoothies,
-      ingredients: ingredients
+      ingredients: ingredients,
+      error: '',
+      chosenIngredients: [],
     }
   }
 
+  fetchData = () => {
+    fetch('http://whateverly-datasets.herokuapp.com/api/v1/smoothies')
+    .then(response => response.json())
+    .then(result => {
+      this.setState({
+        smoothies: result.smoothies
+      })
+    })
+    .catch(error => {
+      this.setState({error: error.message})
+    })
+    fetch('http://whateverly-datasets.herokuapp.com/api/v1/ingredients')
+      .then(response => response.json())
+      .then(result => {
+        this.setState({
+          ingredients: result.ingredients
+        })
+      })
+      .catch(error => {
+      this.setState({error: error.message})
+      })
+  }
```

Comment:
Feel free to explore `Promise.all()` for chaining multiple fetch calls if you feel you can take on a bit more.  Not necessary, but looks like you're all in a good spot right now based on the code I've seen.